docs/gallery/gallery/layout/faceting.jl

```diff
@@ -58,6 +58,14 @@ df2 = (x=[0, 1], y=[0.5, 0.5], i=fill("a", 2), j=fill("e", 2))
 layers = data(df1) * visual(Scatter) + data(df2) * visual(Lines)
 fg = draw(layers * mapping(:x, :y, col=:i, row=:j))
 
+# ## Caveats
+# The faceting variable must be non-numeric. If the source is numeric, you can convert it with `nonnumeric`.
+
+df = (x=rand(100), y=rand(100), l=rand([1, 2, 3, 4, 5], 100))
+plt = data(df) * mapping(:x, :y, layout=:l => nonnumeric)
+draw(plt)
+
+# `plt = data(df) * mapping(:x, :y, layout=:l)` would ignore the layout argument.
```

Comment:
```suggestion
```

I'm removing this part because what happens exactly is likely to change (it will probably start throwing an error in future versions).